src/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,26 @@
+import { BiFile } from 'react-icons/bi'
+
+export default function SideNav() {
+  return (
+    <div className="bg-[#03111b] w-[305px] text-[#FFFFFF]">
```

Comment:
background color already defined in tailwind config file, e.g. bg-background, and try to use tailwind preset properties first, like text-white instead

Reply:
Cool~